tests/unit/containers/find-offer/create-subject/CreateSubject.spec.jsx

```diff
@@ -27,28 +27,32 @@ describe('CreateSubjectModal container', () => {
 
   it('should render component', () => {
     const title = screen.getByText('categoriesPage.newSubject.title')
-
     expect(title).toBeInTheDocument()
   })
-  it('should change autocomplete by choosing option', () => {
-    const categoryInput = screen.getByLabelText('offerPage.labels.category')
 
-    expect(categoryInput).toBeInTheDocument()
-
-    fireEvent.click(categoryInput)
-    fireEvent.change(categoryInput, {
-      target: { value: 'Category 1' }
-    })
-    fireEvent.keyDown(categoryInput, { key: 'Escape' })
+  it('should allow selecting and clearing category input', () => {
+    const categoryInput = screen.getByLabelText('offerPage.labels.category')
 
+    fireEvent.change(categoryInput, { target: { value: 'Category 1' } })
     expect(categoryInput.value).toBe('Category 1')
 
-    fireEvent.click(categoryInput)
-    fireEvent.change(categoryInput, {
-      target: { value: '' }
+    fireEvent.change(categoryInput, { target: { value: '' } })
+    expect(categoryInput.value).toBe('')
+  })
+
+  it('should submit form with correct values', async () => {
```

Comment:
It's not async function currently
```suggestion
  it('should submit form with correct values', () => {
```